integration/query_projection_test.go

```diff
@@ -252,3 +252,35 @@ func TestQueryProjectionErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestQueryProjectionSuccess(t *testing.T) {
+	t.Parallel()
+	ctx, collection := setup.Setup(t, shareddata.Scalars, shareddata.Composites)
+	for name, tc := range map[string]struct { //nolint:vet // used for testing only
+		filter     bson.D // required
+		projection any    // required
+
+		skip       string              // optional, skip test with a specified reason
+	}{
+		"QueryProjection_v": {
```

Comment:
We use capitalization style for test case name https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md#integration-tests-naming-guidelines